packages/strapi-design-system/src/themes/shadows.js

```diff
@@ -1,6 +1,8 @@
 export const shadowTokenObject = {
   shadow: {
     filterShadow: '0px 1px 4px rgba(33, 33, 52, 0.1)',
+    focus:
```

Comment:
cc @maevalienard should we modify the places where we used the focus by this thing?

Reply:
And for the focus state, I would highly recommend something strongly visible, like a border, a dashed border or something like that. Most people that will heavily rely on this will have visual impairments, I think

Reply:
I don't know what this is 😂

Reply:
Let's close this PR for now then :p